src/SplitFlapDisplay/index.tsx

```diff
@@ -23,12 +26,15 @@ export interface SplitFlapDisplayProps extends React.HTMLProps<HTMLDivElement> {
 const defaultProps = {
   background: '#000000',
   borderColor: '#dddddd',
+  borderRadius: '10px', // Default border radius
   borderWidth: '1px',
   characterSet: NUMERIC,
   characterWidth: '1rem',
+  fontFamily: 'Arial, sans-serif', // Default font family
   fontSize: '1rem',
   minLength: 5,
   padDirection: 'left',
+  margin: '5px', // Default padding
```

Comment:
`1px`, please, also comment could be more descriptive in regards to what exactly is being spaced

Reply:
gotcha, done